includes/class-webmention-homepage-widget.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+class Homepage_Webmentions extends WP_Widget {
+
+	/**
+	 * Register widget with WordPress.
+	 */
+	public function __construct() {
+		parent::__construct(
+			'Webmention_Homepage_Webmentions_Widget',
+			esc_html__( 'Homepage Webmentions Widget', 'webmention' ),
+			[
+				'classname'   => 'webmention_homepage_webmentions_widget',
+				'description' => esc_html__( 'A widget that allows you to display your homepage webmentions.', 'webmention' ),
+			]
+		);
```

Comment:
WordPress naming standards suggest a correlation between filename and classname

Reply:
Addressed in 4fb9cee